js/ui/overlay/ui.overlay.js

```diff
@@ -1540,6 +1458,16 @@ const Overlay = Widget.inherit({
             case 'wrapperAttr':
                 this._renderWrapperAttributes();
                 break;
+            case 'dragAndResizeArea':
+                if(this.option('resizeEnabled')) {
+                    this._resizable.option('area', this._getDragResizeContainer());
+                }
+                if(this.option('dragEnabled')) {
+                    this._draggable.updateDragContainer(this._getDragResizeContainer());
+                    this._draggable.updateOutsideMultiplayer(this.option('dragAndResizeArea').outsideMultiplayer);
```

Comment:
```suggestion
                    this._draggable.updateOutsideMultiplayer(value?.outsideMultiplayer);
```

Reply:
value can contain `container` field only, but i guess we should save old `outsideMultiplayer` value

Reply:
not actual anymore, now these options are separeted